src/config.js

```diff
@@ -50,7 +50,7 @@ global.config = {
   },
 
   memory: {
-    segments: 20,
+    segments: 10,
```

Comment:
they should be enabled by default, and set to 10, i suggest

Reply:
Since they were introduced I constantly ran into errors (https://github.com/TooAngel/screeps/pull/503) Especially that the size of a single segment was exceeded. This was never addressed or fixed in any way and I nearly lost all of my rooms.

As long as the segment implementation can't handle these cases properly, I don't think it is a good idea to use them as default.
Also due to the isolated VMs the global cache is way more reliable, so I would make first more use of that and see what we actually need to store in the Memory/Segments.